ammo-be/src/routers/bulletRouter/__tests__/ws.test.ts

```diff
@@ -78,4 +79,61 @@ describe('[WS] BulletRouter', () => {
             done();
         });
     });
+
+    it('The recording state must be initialized to true', () => {
+        expect(isRecording).toBe(true);
+    });
+
+    it('Should change the recording state to false', (done) => {
+        socket.emit('toggleRecord', (result: boolean) => {
+            expect(result).toBe(false);
+            expect(isRecording).toBe(false);
+            done();
+        });
+    });
+
+    it('Should not be emitting if the recording state is false', async (done) => {
+        const emitSpy = jest.spyOn(ioServer, 'emit');
+
+        await request(app)
+            .post('/')
+            .send({ data: connectorRequestMock })
+            .expect(200);
+
+        expect(emitSpy).toBeCalledTimes(0);
```

Comment:
Sweet. Is the done() needed though?

Reply:
Yup, once specified as parameter, we need to invoke, otherwise jest never end the test